services/server/src/server/services/workers/verificationWorker.ts

```diff
@@ -63,16 +67,41 @@ const initWorker = () => {
   vyper = new VyperLocal(Piscina.workerData.vyperRepoPath);
 };
 
-export async function verifyFromJsonInput({
+async function runWorkerFunction<T extends VerificationWorkerInput>(
```

Comment:
Some descriptive naming would be helpful, I had to dig in what this does
```suggestion
async function runWorkerFunctionWithContext<T extends VerificationWorkerInput>(
```